samples/angular-todo/src/app/views/login/login.component.html

```diff
@@ -11,18 +11,25 @@
 <div class="cstm_container_v-centered container-fluid d-flex align-items-center">
   <div class="w-100">
     <app-back-home></app-back-home>
+    <ng-container>
+      <div class="card shadow-sm p-5 mb-2 w-100">
+        <div class="cstm_form-icon align-self-center mb-3">
+          <ng-container *ngIf="!isWebAuthn; else fingerPrint">
```

Comment:
Do we need to make the same change to `register.component.html` and `register.component.ts`? Usually, the WebAuthn would be optionally registered in the registration flow. So, both the login flow and the registration flow should handle WebAuthn, which is why I discouraged the core logic from being in this view. This logic looks like it just handles icon changes, which is great, but it would be nice if this same treatment was on the register flow.

Reply:
this is only intended to switch the key-icon to the finger-print icon? is it necessary for the registration? we dont just fill out a form when registering? may be wrong. It culd be easily added tho, just asking.
<img width="411" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/cf5429bd-807d-4b24-b628-727671b76601">
==========================
<img width="384" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/7db4ebde-de69-4543-bbd5-6ea9e69c00f1">

Reply:
this is what i meant
<img width="434" alt="image" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/1338bc33-aa70-4534-a9af-7c270ad90060">